index.html

```diff
@@ -70,6 +70,188 @@ <h1>MUHYIDEEN ELIAS</h1>
+        <div class="two-fingers"><img src="./images/two-fingers.svg" alt="two-fingers"></div>
+        <div class="twitter"><img src="./images/twitter.svg" alt="twitter"></div>
+        <div class="medium"><img src="./images/medium.svg" alt="medium"></div>
+      </div>
+      <div class="arrow-card">
+        <div class="arrow"><img src="./images/arrow.svg" alt="work"></div>
+        <span>Work</span>
+      </div>
+    </div>
+
+  </section>
+  <section class="section-4">
+    <div class="my-recent-work">
+      <h4>MY RECENT WORK</h4>
+    </div>
+    <div class="breakage">
+      <hr>
+    </div>
+    <div class="project">
+
+      <div class="featured-project">
+        <div class="recipe-card">
+          <img src="./images/project-img/drop.svg" alt="recipe">
+        </div>
+        <div class="details-card">
+          <div class="project-header">
+            <h4>Natural Drop</h4>
+          </div>
+          <div class="project-body">
+            <span>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint laboriosam minus sunt expedita voluptatum
+              est repudiandae nesciunt soluta consectetur, iure quia, provident nostrum optio atque quas itaque ut
+              reprehenderit omnis?
+            </span>
+            <div class="programming-languages">
+              <ul>
+                <li class="css">css</li>
+                <li class="html">html</li>
+                <li class="boostrap">boostrap</li>
+                <li class="ruby">Ruby</li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <span>See Project</span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="featured-project">
+        <div class="recipe-card">
+          <!-- <img src="./images/project-img/flowers.svg" alt="recipe"> -->
+        </div>
+        <div class="details-card">
+          <div class="project-header">
+            <h4>Bloom Culture</h4>
+          </div>
+          <div class="project-body">
+            <span>
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Reiciendis, quas nisi porro recusandae vel est
+              officiis soluta ab facere quidem delectus fugiat modi culpa sequi repellendus molestiae perspiciatis
+              aspernatur atque!
+            </span>
+            <div class="programming-languages">
+              <ul>
+                <li class="css">css</li>
+                <li class="html">html</li>
+                <li class="bootrap">boostrap</li>
+                <li class="ruby">Ruby</li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <span>See Project</span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="featured-project">
+        <div class="recipe-card">
+          <img src="./images/project-img/yoga.svg" alt="recipe">
```

Comment:
[OPTIONAL] Please correct your image path because some browser make difference between UPPERCASE and lowercase

instead of yoga.svg => Yoga.sgv

<img width="845" alt="image" src="https://user-images.githubusercontent.com/7243132/170833781-cff66359-8256-41b3-9896-eae7b0285d31.png">

Reply:
thank you very much @jaflih, fixed that